yoti-sdk-api/src/main/java/com/yoti/api/client/qrcode/policy/DynamicPolicyBuilder.java

```diff
@@ -0,0 +1,84 @@
+package com.yoti.api.client.qrcode.policy;
+
+import java.util.ServiceLoader;
+
+public abstract class DynamicPolicyBuilder {
+
+    public static final DynamicPolicyBuilder newInstance() {
```

Comment:
Yes, I want to prevent it being overriden, so it is _final_.
Any subclasses of this class should be internal to the _impl_ module - they cannot be instantiated directly.  The only way to acquire an instance of this builder (or in fact a subclass of it) is via this static method.